packages/discord.js/src/util/Transformers.js

```diff
@@ -9,6 +10,7 @@ const snakeCase = require('lodash.snakecase');
  */
 function toSnakeCase(obj) {
   if (typeof obj !== 'object' || !obj) return obj;
+  if (isJSONEncodable(obj)) return obj.toJSON();
```

Comment:
I feel like this check should be done beneath line 14 where it checks `instanceof Date` as that I believe is JSON-encodeable, no?

Reply:
I was not aware of that. Fixed.